lambda_functions/vetext_incoming_forwarder_lambda/requirements-lambda.txt

```diff
@@ -0,0 +1 @@
+requests==2.28.0
```

Comment:
Just curious - should we use the latest version here (2.28.1) or use a compatible flag for this version? ~=2.28.0?